nb-apis/adapters-api/src/main/java/io/nosqlbench/adapters/api/activityimpl/uniform/opwrappers/DryCycleOp.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2022-2024 nosqlbench
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Add more testing for this commit

Reply:
Added #2064 to request better documentation and examples of this class. I think tests for these capabilities need to be shown in a user-accessible way, so should be part of the scenario verification layer.